lib/ui/view/channel/channel_members/channel_members_list_model.dart

```diff
@@ -67,6 +67,6 @@ class ChannelMembersModel extends BaseViewModel {
   }
 
   void goBack() {
-    NavigationService().back();
+    navigator.back();
```

Comment:
rename to navigation service

Reply:
It is still the same thing.

navigator = locator<NavigationService>();
    It's still navigation service that was used.
    The test kills returning a null value if NavigationService is used directly inside of navigator.

Reply:
I never meant NavigationService itself.

I meant change the variable name to ```_navigationService``` for uniformity